pkg/operation/botanist/monitoring.go

```diff
@@ -49,8 +49,7 @@ import (
 )
 
 const (
-	secretNameIngressOperators = v1beta1constants.SecretNameObservabilityIngress
-	secretNameIngressUsers     = v1beta1constants.SecretNameObservabilityIngressUsers
+	secretNameIngressUsers = v1beta1constants.SecretNameObservabilityIngressUsers
```

Comment:
Eventually @wyb1 and I decided to continue using the SecretNameObservabilityIngressUsers secret name to avoid changing the secret for end users as a side effect of this PR.